code/src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import '../style/button.css';
```

Comment:
I am excited that you experimented putting css file for each component. I have not tried it myself yet. This method looks neat and seems practical when the project becomes big and complicated. Nice job!